File: fizzbuzz/fizzbuzz.ml
let fizzbuzz n =
        match (n mod 3, n mod 5) with
        | (0, 0) -> "fizzbuzz"
        | (0, _) -> "fizz"
        | (_, 0) -> "buzz"
        | (_, _) -> string_of_int n

let () = for n = 1 to 100 do
        Printf.printf "%s\n" (fizzbuzz n)
done

File: fizzbuzz/fizzbuzz.rs
pub fn main() {
    for n in 1..=100 {
        print!("{}\n", fizzbuzz(n));
    }
}

fn fizzbuzz(n: u32) -> String {
    match (n % 3, n % 5) {
        (0, 0) => "fizzbuzz".to_string(),
        (0, _) => "fizz".to_string(),
        (_, 0) => "buzz".to_string(),
        (_, _) => n.to_string(),
    }
}
